src/main/resources/crafter/core/core.properties

```diff
@@ -132,5 +132,9 @@ crafter.core.rest.management.authorizationToken=defaultManagementToken
 ################################################################################
 # BLOB STORE PROPERTIES
 ################################################################################
+# The extension for the blob files
 crafter.blob.extension=blob
-crafter.blob.configuration=/config/studio/blob-stores-config.xml
+# The module for the configuration file
+crafter.blob.configuration.module=studio
```

Comment:
Why is the module Studio instead of Engine?

Reply:
The module is just used for building the path, it has always been in `/config/studio`. The same applies for aws, box & webdav profiles

Reply:
Got it, thanks.